tests/test-cases.js

```diff
@@ -694,6 +694,24 @@ var PARSE_TESTS = [
 		}
 	},
 	{
+		description: "Dynamic typing by indices can be determined by function",
+		input: '001,002,003',
+		config: { dynamicTyping: function(field) { return (field % 2) === 0; } },
+		expected: {
+			data: [[1, "002", 3]],
+			errors: []
+		}
+	},
+	{
+		description: "Dynamic typing by headers can be determined by function",
```

Comment:
The code is the same using integers and strings, so no need to test anything else. For documentation use the gh-pages branch which contains the documentation.